docs/DeveloperGuide.md

```diff
@@ -113,7 +113,15 @@ and further isolates the area of concern for which errors are most prone. In add
 methods if required. If possible, additional methods should continue to depend on the first `getString` method.
 
 ### Menu (dishes) Management
+## Overview
+![](diagrams/class/DishManagementOverview.png)
 
+## Dish
+
+`Dish` is basically consist of `name` and `price` and corresponding modification methods.
```

Comment:
I think "`Dish` consists of `name` and `price` and corresponding modification methods." might be better